DQM/L1TMonitorClient/python/L1TStage2EmulatorMonitorClient_cff.py

```diff
@@ -53,11 +53,11 @@
                       + l1tStage2OMTFEmulatorClient
                       + l1tStage2EMTFEmulatorClient
                       + l1tStage2EmulatorEventInfoClient
-                      + l1tStage2uGTEmulatorClient
+                      #+ l1tStage2uGTEmulatorClient
                         )
 
 l1tStage2EmulatorMonitorClient = cms.Sequence(
-                        l1TStage2EmulatorQualityTests +
-                        l1TStage2EmulatorClients
+                        l1TStage2EmulatorQualityTests #+
+                      #  l1TStage2EmulatorClients
```

Comment:
Do not comment out this line if you only want to drop the uGT data vs. emulator plots. Commenting out this will remove the plots for all subsystems.